FinalProject/gs-rest-service-master/complete/src/main/java/services/CartService.java

```diff
@@ -0,0 +1,18 @@
+package services;
+
+import java.sql.SQLException;
+import java.util.List;
+
+import bo.Cart;
+import bo.CartItem;
+import bo.User;
+
+public interface CartService {

```

Comment:
Minor: it's not necessary to set access modifiers in method here because always are public in an interface.